wcfsetup/install/files/lib/system/template/TemplateEngine.class.php

```diff
@@ -694,6 +694,8 @@ public function disableSandbox()
      * @param array $variables
      * @param bool $sandbox enables execution in sandbox
      * @return  string
+     *
+     * @deprecated 6.2 use `render()` instead
```

Comment:
Please forward all calls with `$sandbox === true` to `render()` and remove the handling from this method.